pennylane/qchem/openfermion_obs.py

```diff
@@ -161,6 +161,15 @@ def observable(fermion_ops, init_term=0, mapping="jordan_wigner", wires=None):
             openfermion.transforms.bravyi_kitaev(mb_obs), wires=wires
         )
 
+    if mapping == "parity":
+        qubits = openfermion.count_qubits(mb_obs)
+        if qubits == 0:
```

Comment:
When do we expect this to happen? 🤔

Reply:
Sometimes one of the x,y,z components of the dipole moment operator are 0, when mapped this should return I(0). For openfermion, when such an operator is passed to count_qubits, it gives 0.